docs/admin/ha-master-gce.md

```diff
@@ -107,7 +107,7 @@ This operation may be sped up by migrating etcd data directory, as described [he
 
 ## Implementation notes
 
-![](ha-master-gce.png)
+![ha-master-gce](http://kubernetes.io/images/docs/ha-master-gce.png)
```

Comment:
Right now, the image is not showing up in the Netlify preview. You should use a relative path for the URL instead of an absolute one, i.e.:

!\[ha-master-gce](/images/docs/ha-master-gce.png)